_v1.1/docs/admin/cluster-large.md

```diff
@@ -40,7 +40,7 @@ To avoid running into cloud provider quota issues, when creating a cluster with
 
 ### Addon Resources
 
-To prevent memory leaks or other resource issues in [cluster addons](https://releases.k8s.io/v1.1.0/cluster/addons) from consuming all the resources available on a node, Kubernetes sets resource limits on addon containers to limit the CPU and Memory resources they can consume (See PR [#10653](http://pr.k8s.io/10653/files) and [#10778](http://pr.k8s.io/10778/files)).
+To prevent memory leaks or other resource issues in [cluster addons](https://releases.k8s.io/HEAD/cluster/addons) from consuming all the resources available on a node, Kubernetes sets resource limits on addon containers to limit the CPU and Memory resources they can consume (See PR [#10653](http://pr.k8s.io/10653/files) and [#10778](http://pr.k8s.io/10778/files)).
```

Comment:
Looks like the script unversions all files that are not included in k8s.io: https://github.com/kubernetes/kubernetes/blob/gh-pages/_tools/release_docs/main.go#L66